package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "tina-gatsby-starter",
+  "version": "1.0.0",
+  "private": true,
+  "description": "tina-gatsby-starter",
+  "author": "Hugo Pernet",
+  "keywords": [
+    "gatsby"
+  ],
+  "scripts": {
+    "dev": "npx tinacms dev -c \"gatsby develop\"",
+    "start": "gatsby develop",
+    "build": "tinacms build --no-client-build-cache && gatsby build",
```

Comment:
`--no-client-build-cache` do we need this anymore?

Reply:
✅Fixed - removed

I tried without it and it works.!